lib/models/different_floor_direction_handler.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:concordia_go/blocs/bloc.dart';
+import 'package:concordia_go/models/direction_handler.dart';
+import 'package:concordia_go/models/direction_request.dart';
+import 'package:concordia_go/models/shortest_path.dart';
+import 'package:concordia_go/models/graph.dart';
+import 'package:concordia_go/models/node.dart';
+import 'package:concordia_go/utilities/concordia_constants.dart' as cc;
+import 'package:concordia_go/widgets/component/google_maps_component.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class DifferentFloorDirectionHandler implements DirectionHandler {
+  DirectionHandler _nextHandler;
+
+  @override
+  void setNext(DirectionHandler handler) {
+    _nextHandler = handler;
+  }
+
+  @override
+  void handle(DirectionRequest request) {
+    if (canHandle(request)) {
+      final String sourceBuildingCode = request.source.building.code + request.source.floor;
+      final String destinationBuildingCode = request.destination.building.code + request.destination.floor;
+
+      final Graph sourceGraph =
+          Graph(sourceBuildingCode, cc.edges[sourceBuildingCode], cc.edge_indices[sourceBuildingCode]);
+      sourceGraph.setNodesFromEdgeIndices(cc.edge_indices[sourceBuildingCode]);
+
+      final Graph destinationGraph =
+          Graph(destinationBuildingCode, cc.edges[destinationBuildingCode], cc.edge_indices[destinationBuildingCode]);
+      destinationGraph.setNodesFromEdgeIndices(cc.edge_indices[destinationBuildingCode]);
+
+      Node escalatorNode;
+      if (int.parse(request.source.floor) > int.parse(request.destination.floor)) {
+        escalatorNode = Node('120000');
```

Comment:
Random Numbers. Need Explanations

Reply:
Same thing as in the last comment, however in this case, it's an escalator. However I agree it might look random. Do you think a comment line explaining the choice of node should be added, or simply making the naming convention in #21 more easily accessible?

Reply:
Comment line or a variable explaining it